addons/mail/static/src/models/notification_group/notification_group.js

```diff
@@ -67,6 +71,31 @@ function factory(dependencies) {
             return `${this.modelName}_${data.id}`;
         }
 
+        /**
+         * Compute the most recent date inside the notification messages.
+         *
+         * @private
+         * @returns {String|undefined}
+         */
+        _computeDate() {
+            const dates = this.notifications.filter((notification) => notification.message && notification.message.date).map(notification => notification.message.date);
+            if (dates.lenght == 0) {
```

Comment:
typo on length, so we don't have any test covering the actual content of a date on a notification group?